performance/performance_firely_server.rst

```diff
@@ -31,6 +31,11 @@ Search Parameters
 
 When a resource is sent to Firely Server for storage, Firely Server indexes the resource for all the search parameters that are applicable to the resource. The more search parameters are known to Firely Server, the more resources will be used for indexing - both time (for the indexing processing) and storage (for storing the values of each search parameter). This also increases the size of the index tables and indexes, and therefore querying times. Thus, if you know you will only use a portion of the predefined search parameters you can choose to delete the others from the Administration API - see :ref:`conformance` and :ref:`supportedmodel`.
 
+Additionally, these search parameters can be disabled in order to improve performance: 
```

Comment:
I would maybe refrain from using "disabled", the search parameter is still available.
Suggestion: "Additionally, these search parameters can have a large impact on the search performance:"

Reply:
Agreed! I'll refrain from using 'large' as that is dependent on the size of the database but it makes more sense your way.